Cargo.toml

```diff
@@ -1,89 +1,46 @@
+cargo-features = ["profile-rustflags"]
+
 [package]
 name = "bootloader"
-version = "0.10.13"
+version = "0.11.0-alpha"
 authors = ["Philipp Oppermann <dev@phil-opp.com>"]
 license = "MIT/Apache-2.0"
 description = "An experimental x86_64 bootloader that works on both BIOS and UEFI systems."
 repository = "https://github.com/rust-osdev/bootloader"
-edition = "2018"
-build = "build.rs"
+edition = "2021"
+resolver = "2"
```

Comment:
`resolver = 2` is the default in Rust 2021.

Reply:
Ah yes, I'll remove it!

Reply:
Done as part of https://github.com/rust-osdev/bootloader/pull/232/commits/2ca6ebd4d9a8e8b52fa48eaae8dab476b3bee8c9